docs/topics/api/addons.rst

```diff
@@ -306,3 +308,21 @@ a specific add-on by id, slug or guid.
                incompatible  multiprocessCompatible marked as false in the install.rdf.
                     unknown  multiprocessCompatible has not been set.
     =======================  ==========================================================
+
+-------------
+Add-on Policy
+-------------
+
+.. _addon-policy:
+
+This endpoint allows you to fetch an add-on privacy policy and EULA.
+
+.. http:get:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/policy/
```

Comment:
I won't go so far as to say they should be different endpoints because I get how they're very linked things, but I think the API/URL should reflect the fact that it returns two things. eg:

`/api/v3/addons/addon/(int:id|string:slug|string:guid)/eula-policy/`

or something like that.